src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let result = initialValue;
+    let startValue = 0;
+
+    if (arguments.length < 2) {
+      result = this[0];
+      startValue = 1;
+    }
```

Comment:
it's perfect case to use ternary operator, f.e: 
`const startValue = arguments.length < 2 ? 1 : 0;`